.github/workflows/prototype-tests-linux-gpu.yml

```diff
@@ -0,0 +1,97 @@
+name: Prototype tests on Linux
+
+on:
+  pull_request:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python-version:
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
```

Comment:
nit: this can be a list instead, as we've done for the below 2 inputs.

Reply:
I personally prefer the flow style. Meaning I use

```yaml
- foo
- bar
```

everywhere enhanced readability. The only exception I make is for lists of a single entry that are only there because a list is required. For them I use the `["baz"]` syntax.